prow/plugins/plugins.go

```diff
@@ -710,6 +716,12 @@ func (c *Configuration) setDefaults() {
 			},
 		}
 	}
+
+	for name, spec := range c.ConfigUpdater.Maps {
+		spec.Namespaces = append(spec.AdditionalNamespaces, spec.Namespace)
```

Comment:
This may modify `spec.AdditionalNamespaces` I don't think that is safe?

Reply:
If we don't use it downstream, why?

Reply:
However, it is subtle so I can change the ordering